bin/docker/kafka_bridge_run.sh

```diff
@@ -70,4 +67,4 @@ if [ "$FIPS_MODE" = "disabled" ]; then
 fi
 
 # starting Kafka Bridge with final configuration
-exec /usr/bin/tini -s -w -e 143 -- "${MYPATH}"/../kafka_bridge_run.sh --config-file=/tmp/kafka-bridge.properties "$@"
+exec /usr/bin/tini -s -w -e 143 -- "${MYPATH}"/../kafka_bridge_run.sh "$@"
```

Comment:
you need to pass the configuration file as it is but using the `-Dquarkus.config.locations=` system property